### DISCUSSION TITLE
Lucid ORM Feature add

### DISCUSSION THREAD

**Discussion by anmirza** [adonisjs/core#4628] [Ideas]
Hi, Guys hope you are doing well, 
i just want to ask something is there any method to get the last or the latest entry of the table
means just like .first() there should be a .last() method,
its just a suggestion

e.g: Modal.query().where('field',fieldValue).last()

in stead i have to do this:

Modal.query().where('field',fieldValue).orderBy('id','desc').first()

**Comment by RomainLanz** [comment]
Hey @anmirza786! 👋🏻 

We cannot define magically what order needs to be done to decide which item is the last one.
The `first()` helper only defines a `LIMIT` of 1 on whatever query you create.

**Reply by anmirza** [reply]
@RomainLanz thanks for answering my question. I thought it could be convenient to have a last helper function that also limits it to one and order it desc by it id.

But

I got your point.

Thanks

**Reply by RomainLanz** [reply]
> that also limits it to one and order it desc by it id.

Not all ids are sortable.
